editor/editor_quick_open.cpp

```diff
@@ -32,12 +32,18 @@
 
 #include "core/os/keyboard.h"
 #include "editor/editor_node.h"
+#include "editor/editor_scale.h"
+
+static RBMap<StringName, Size2i> sizes;
 
 void EditorQuickOpen::popup_dialog(const StringName &p_base, bool p_enable_multi, bool p_dontclear) {
 	base_type = p_base;
 	allow_multi_select = p_enable_multi;
 	search_options->set_select_mode(allow_multi_select ? Tree::SELECT_MULTI : Tree::SELECT_SINGLE);
-	popup_centered_clamped(Size2i(600, 440), 0.8f);
+	if (!sizes.has(p_base)) {
+		sizes.insert(p_base, Size2i(600, 440));
+	}
+	popup_centered_clamped(sizes[p_base] * EDSCALE, 0.8f);
```

Comment:
The size is integer vector, so multiplication does not work for non-integer scales (e.g. 1.5).

Reply:
There are lots of places in the engine where this is already used, so I guess it doesn't matter:

![image](https://user-images.githubusercontent.com/3036176/203259440-3f792e9c-706d-4178-8b77-68d53d5f810b.png)

Reply:
Yes, the `sizes` map added in this PR uses Size2i. All examples you showed use Size2, so they are correct.

Reply:
Ah, sorry, I didn't notice...

Reply:
@KoBeWi Changed type to Size2